DNN Platform/DotNetNuke.Web/InternalServices/SearchServiceController.cs

```diff
@@ -37,6 +37,7 @@ public class SearchServiceController : DnnApiController
         private const string ModuleTitleCacheKey = "SearchModuleTabTitle_{0}";
         private const CacheItemPriority ModuleTitleCachePriority = CacheItemPriority.Normal;
         private const int ModuleTitleCacheTimeOut = 20;
+        private const int MaximumPageSize = 100;

```

Comment:
100 seems a little small to me for an absolute limit. If we made this configurable via a host setting, I'm fine with 100 as a default.

Reply:
I agree with you but when the user searches in DNN, shows a maximum of 100 results per page.

![image](https://github.com/dnnsoftware/Dnn.Platform/assets/58891213/d2828e99-225f-4aa1-a5bb-56a7d4086389)